lib/controllers/characters.js

```diff
@@ -0,0 +1,46 @@
+const { Router } = require('express')
+const Character = require('../models/Character')
+const pool = require('../utils/pool.js')
+
+module.exports = Router()
+  .post('/', async (req, res) => {
+    const character = await Character.insert({
```

Comment:
Character.insert(req.body) also works because req.body is the object that has all of these key value pairs. It's nice to use when the table has a lot of columns like this one does